exercises/package.json

```diff
@@ -7,34 +7,31 @@
     "@reduxjs/toolkit": "^1.4.0",
     "@salsita/react-api": "^1.0.0",
     "@salsita/react-core": "^1.0.0",
-    "@salsita/react-crud": "^1.0.0",
+    "@salsita/react-crud": "^1.0.1",
```

Comment:
do we need those @salsita/react-XXX deps in this file?

Reply:
Removed unused dependencies 7073a20